sponsorship/src/main/java/com/kyriba/conference/sponsorship/service/EmailClientAsync.java

```diff
@@ -0,0 +1,5 @@
+package com.kyriba.conference.sponsorship.service;
+
+public interface EmailClientAsync extends BasicEmailClient
```

Comment:
Strange approach with interface as marker only for extension

Reply:
Fixed